src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 198px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+    background-size: contain;
+  }
+
+  &__item {
+    font-size: 12px;
+    text-align: left;
```

Comment:
you don't have any element with class `card__item`, so those styles are not used